File: 05_Inverted hollow half pyramid/inverted_hollow_half_pyramid.py
row = int(input("Enter rows: "))

for i in range(row, 0, -1):
    for j in range(0, i):
        if(i==row or i==1 or j==0 or j==(i-1)):
            print("*", end=" ")
        else:
            print(" ", end=" ")
    print("")

'''
OUTPUT:

* * * * * * 
*       *
*     *
*   *
* *
*

'''

File: 05_Inverted hollow half pyramid/inverted_hollow_half_pyramid.c
#include<stdio.h>

int main(int argc, char const *argv[])
{
    int rows;

    printf("Enter rows: ");
    scanf("%d", &rows);

    for(int i=rows; i>0; i--){
        for(int j=0; j<i; j++){
            if(i==rows || i==1 || j==0 || j==(i-1)){
                printf("* ");
            }else{
                printf("  ");
            }
        }
        printf("\n");
    }
    return 0;
}

/*
OUTPUT

* * * * * * 
*       *
*     *
*   *
* *
*

*/